src/platform.js

```diff
@@ -41,12 +41,12 @@ exports = module.exports = {
 	* `true` if the platform has native single-finger [events]{@glossary event}.
 	* @public
 	*/
-	touch: Boolean(('ontouchstart' in window) || window.navigator.msMaxTouchPoints),
+	touch: Boolean(('ontouchstart' in window) || window.navigator.msMaxTouchPoints || window.navigator.maxTouchPoints),
```

Comment:
Ah yes, my mistake - somehow was considering these to be mutually exclusive. I think the existing check is good, and we should change the other check to `> 1`.